content/account/dashboard/api.textile

```diff
@@ -0,0 +1,56 @@
+meta_description: “Manage Ably API keys by creating, updating, setting restrictions, and exploring integration options.”
+meta_keywords: "API keys, Ably, create key, update key, key restrictions, integration, webhooks, authentication"
+---
+
+The API keys tab lists all API keys associated with your account and provides information on each key's capabilities and restrictions. You can "create a new API key":#create and manage an API key.
+
+h2(#create). Create a new API key
+
+Click  *create new API key* to create a new API key. During the _create_ process, you can:
+* Assign a friendly name.
+* Give the new API key a descriptive name (e.g. chat app key) so it is easy to identify later.
+
+h2(#manage). Manage an API key
+
+To manage an API key, you set capabilities, define resource restrictions, use revocable tokens for security, and adjust key settings as needed.
+
+h3. Capabilities
+
+Capabilities provide permissions required for managing message flow, user presence, notifications, channel information, and access controls, these depend on what you require the API key to have access to:
+
+|_. Capability |_. Description |
+| *Publish* | Enable clients to publish messages to channels. |
+| *Subscribe* | Enable clients to receive messages and presence state changes. |
+| *History* | Enable permission to retrieve message and presence history. |
+| *Presence* | Enable clients to register presence on a channel. |
+| *Channel metadata* | Enable querying of channel metadata. |
+| *Push admin and push-subscribe* | Manage and subscribe to push notifications. |
+| *Statistics* | Access usage statistics. |
+| *Privileged headers* | Set privileged headers, such as skipping channel rules. |
+
+h3. Set resource restrictions
+
+Set resource restrictions to control access to channels and queues, ranging from unrestricted access to specific, rule-based permissions:
+
+|_. Restriction |_. Description |
+| *None* | No restrictions; access any channel or queue. |
+| *Only channels* | Access any channel but not queues. |
+| *Only queues* | Access any queue but not channels. |
+| *Selected channels and queues* | Specify explicit rules for access. |
+
+h3. Revocable tokens
+
+Revocable tokens enhance security by allowing shorter token lifetimes and the ability to revoke tokens issued via the API key.
+
+|_. Option |_. Description |
+| *Revocable tokens* | Implement security measures by setting shorter token lifetimes and enabling the ability to revoke tokens issued by the API key. |
+
+h3. Change your API key settings
+
+Click *settings* on the required API key to change its settings. The same settings apply as when creating a new API key.
+
+h2. Integrations
+
+You can configure and manage "integration":/docs/general/integrations rules, set up webhooks to connect Ably with other systems, and explore options for sending realtime data to third-party services like Datadog. Additionally, it provides access to tools for managing API keys, queues, and notifications.
```

Comment:
This needs some work, as not sure about the API key management. Also we can ingest data with integrations too, though that isn't technically managed in the dashboard...